include/TPP/Passes.h

```diff
@@ -100,7 +100,6 @@ std::unique_ptr<OperationPass<ModuleOp>>
 createTransformDialectInterpreterPass();
 std::unique_ptr<OperationPass<func::FuncOp>> createConvertPerfToLoopsPass();
 std::unique_ptr<OperationPass<ModuleOp>> createConvertPerfToFuncPass();
-std::unique_ptr<OperationPass<func::FuncOp>> createCombineTppPass();
```

Comment:
Please don't remove it in this PR, since it's not used in the "linalg-to-xsmm" pass anyway, it won't interfere with your work.

Reply:
reverted